### DISCUSSION TITLE
Phoenix presence through supabase realtime to show connected users?

### DISCUSSION THREAD

**Discussion by nvsd** [supabase/supabase#5544] [Questions] [answered]
We currently have a phoenix server that handles chat messages in our web app that I'm trying to replace with supabase realtime. 
I was able to replace basic functionality pretty quickly (and in 20 or so lines of code!), but I now need to replace a feature that shows who's currently in the chat room. I used phoenix presence to do this previously.

Is this possible to replicate using supabase, and is this planned for the future if not?

**Comment by w3b6x9** [comment, marked as answer]
@nvsd glad to hear you're using phoenix already! we're in the process of integrating phoenix presence into supabase and will be released very soon.

**Reply by ermincelikovic** [reply]
As soon as the next launch week? :)

**Reply by w3b6x9** [reply]
possibly... 😉

**Reply by nvsd** [reply]
Awesome, thanks for letting me know!

**Comment by EdgarsLv** [comment]
when is this going to be live? 
https://supabase.com/blog/2022/04/01/supabase-realtime-with-multiplayer-features

**Reply by w3b6x9** [reply]
@EdgarsLv We're going to slowly release it to those who sign up for the waitlist. You can add yourself: https://multiplayer.dev
